scripts/hash-unity-files.sh

```diff
@@ -7,6 +7,7 @@ find ./unity-renderer -type f \
     \( -not -path '*Library*' \) \
     \( -not -path '*browser-interface*' \) \
     \( -not -path '*node_modules*' \) \
+    \( -not -path '*Sentry/SentryOptions.asset' \) \
```

Comment:
Looks like this was deleted on https://github.com/decentraland/unity-renderer/pull/4334

Reply:
My bad, sorry!